config/static_caching.php

```diff
@@ -125,4 +125,16 @@
 
     'warm_queue' => null,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Queue Connection
+    |--------------------------------------------------------------------------
+    |
+    | Here you may define the queue connection name that
+    | will be used when warming the static cache.
+    |
+    */
+
+    'queue_connection' => env('STATAMIC_STATIC_WARM_QUEUE_CONNECTION'),
```

Comment:
I wonder if it might be worth changing this to `warm_queue_connection` to stay consistent with `warm_queue` above?

Reply:
Also, I wonder if it might be worth falling back to the default connection here so we don't need to do that in the `StaticWarm` command:

```suggestion
    'warm_queue_connection' => env('STATAMIC_STATIC_WARM_QUEUE_CONNECTION', config('queue.default')),
```